<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>员工后台管理系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="photo.qrc">
    <normaloff>:/2.png</normaloff>:/2.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 主界面 */
QMainWindow#MainWindow
{
border-image: url(:/1.png);
}

/* 文本框 */
QLineEdit#codeEdit,QLineEdit#userEdit
{
 border: none; /* 移除所有边框 */
    border-bottom: 1px solid #ffffff; /* 下边框颜色为白色 */
    
    background-color: rgba(0, 0, 0, 0); /* 透明背景 */
    color: #ffffff; /* 字体颜色为白色 */


}
/* 
QPushButton{

border-radius: 5px;
background-color: rgb(85, 170, 255);
}
悬停 
 QPushButton:hover 
{
 background-color: rgb(100, 200, 255);
 }
按下 
QPushButton:pressed 
{
 background-color: rgb(50, 120, 200);
}
*/

/* 窗口控件 */
QWidget#widget
{
	background-color: rgba(0, 0, 0,0.6);
	border:none;
	
}
/* 标签 */
QLabel#toplabel,QLabel#datalabel,QLabel#timelabel
{
	color: rgb(255, 255, 255);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>400</width>
      <height>260</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="userEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>220</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1996</string>
     </property>
     <property name="placeholderText">
      <string>用户名：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="codeEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>220</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>123456</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>221</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>非本店员工禁止入内</string>
     </property>
    </widget>
    <widget class="QPushButton" name="logButton">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>200</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Agency FB</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QLabel" name="toplabel">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>280</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>员工管理系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="datalabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>161</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>00/00/00</string>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>151</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>00：00：00</string>
    </property>
   </widget>
   <widget class="RollText" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>399</y>
      <width>801</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RollText</class>
   <extends>QWidget</extends>
   <header location="global">rolltext.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="photo.qrc"/>
 </resources>
 <connections/>
</ui>
